src/NuGet.Indexing/Querys/QueryBoostingContext.cs

```diff
@@ -49,20 +49,15 @@ public static QueryBoostingContext Load(string fileName, ILoader loader, Framewo
 
                     var value = serializer.Deserialize<QueryBoostingContext>(reader);
```

Comment:
is it still possible to run search service without any query boosting, or will it fail?